src/System.Net.Security/tests/FunctionalTests/SslStreamSystemDefaultsTest.cs

```diff
@@ -0,0 +1,153 @@
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    using Configuration = System.Net.Test.Common.Configuration;
+
+    public class SslStreamSystemDefaultTest
+    {
+        [Fact]
+        public async Task SslStream_DefaultTlsConfigurationSync_Ok()
+        {
+            using (var test = new SyncTest())
+            {
+                await test.RunTest();
+            }
+        }
+
+        [Fact]
+        public async Task SslStream_DefaultTlsConfigurationApm_Ok()
+        {
+            using (var test = new ApmTest())
+            {
+                await test.RunTest();
+            }
+        }
+
+        [Fact]
+        public async Task SslStream_DefaultTlsConfigurationAsync_Ok()
+        {
+            using (var test = new AsyncTest())
+            {
+                await test.RunTest();
+            }
+        }
+
+        public abstract class TestBase : IDisposable
+        {
+            protected SslStream _clientStream;
+            protected SslStream _serverStream;
+
+            public TestBase()
+            {
+                var network = new VirtualNetwork();
+                var clientNet = new VirtualNetworkStream(network, false);
+                var serverNet = new VirtualNetworkStream(network, true);
+
+                _clientStream = new SslStream(clientNet, false, AllowAnyServerCertificate);
+                _serverStream = new SslStream(serverNet, false, AllowAnyServerCertificate);
+            }
+
+            public async Task RunTest()
+            {
+                X509Certificate2 serverCertificate = Configuration.Certificates.GetServerCertificate();
+                string serverHost = serverCertificate.GetNameInfo(X509NameType.SimpleName, false);
+                X509CertificateCollection clientCertificates = new X509CertificateCollection();
+                clientCertificates.Add(Configuration.Certificates.GetClientCertificate());
+
+                var tasks = new Task[2];
+                tasks[0] = AuthenticateClient(serverHost, clientCertificates, checkCertificateRevocation: false);
+                tasks[1] = AuthenticateServer(serverCertificate, clientCertificateRequired:true, checkCertificateRevocation:false);
+                await Task.WhenAll(tasks);
+
+                
+                if (PlatformDetection.IsWindows && PlatformDetection.WindowsVersion > 10)
+                {
+                    Assert.True( _clientStream.HashAlgorithm == HashAlgorithmType.Sha256 ||
+                                   _clientStream.HashAlgorithm == HashAlgorithmType.Sha384 ||
+                                   _clientStream.HashAlgorithm == HashAlgorithmType.Sha512);
+                }
+            }
+
+            private bool AllowAnyServerCertificate(
+                object sender,
+                X509Certificate certificate,
+                X509Chain chain,
+                SslPolicyErrors sslPolicyErrors)
+            {
+                return true;
```

Comment:
The name says certificate, the test says "whatever".  Should this only return true for untrusted certificate errors (and success)?